website/src/components/Buttons/LikertButtons.tsx

```diff
@@ -25,7 +25,17 @@ export const LikertButtons = ({
     >
       {Object.keys(valueMap).map((value) => {
         return (
-          <Radio key={value} value={value} borderColor="gray.400" data-cy="radio-option" size="md" padding="0.6em" />
+          <Radio
+            key={value}
+            id={`radio-${value}`}
+            value={value}
+            borderColor="gray.400"
+            data-cy="radio-option"
+            size="md"
+            padding="0.6em"
+          >
+            <label htmlFor={`radio-${value}`}>{value}</label>
+          </Radio>
```

Comment:
This change may not be correct. We may have multiple elements with the same id when rendering many `LikertButtons` component